mne_bids/mne_bids.py

```diff
@@ -39,8 +39,11 @@
                  '.fif': 'Elekta', '.gz': 'Elekta', '.pdf': '4D Magnes',
                  '.ds': 'CTF'}
 
+IGNORED_CHANNELS = defaultdict(lambda: [])
+IGNORED_CHANNELS['KIT/Yokogawa'] = ['STI 014']
```

Comment:
currently because I know it will change with annotations, do we make synthetic channels for all systems other than FIF?

Reply:
The annotations stuff in MNE is 6 months away. Let's move faster than that :)

Reply:
lol i know that, my question is, isn't this a concern for all non-FIF systems. They will also have a synthetic channel included if they are saved in their native format and not FIF

Reply:
> do we make synthetic channels for all systems other than FIF?

I think so, because that's the MNE way: read the data from wherever and handle it as if it was a FIF file. So all non-FIF systems will get their STIM 014 channel. Out of experience I only know this von BrainVision data though ...